pgmpy/readwrite/BIF.py

```diff
@@ -0,0 +1,340 @@
+import numpy
+from pyparsing import Word, alphanums, Suppress, Optional, CharsNotIn, Group, nums, ZeroOrMore, OneOrMore, cppStyleComment, Literal, printables
+import re
```

Comment:
@khalibartan I think the import order according to PEP8 should be builtin, then third party. And add a blank line after each block of imports.